src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -282,7 +282,11 @@ public function process()
                 $bMsg .= 'ActionId: ' . $this->_lastActionId . "\r\n" . $aMsg;
                 $event = $this->_messageToEvent($bMsg);
                 $response = $this->findResponse($event);
-                $response->addEvent($event);
+                if ($response === false) {
```

Comment:
is there anything in particular that I can check for you? I'm not very familiar with your library and asterisk protocol, I've just noticed few errors here and there and I decided to contribute back with this fix